action-a/action.yml

```diff
@@ -0,0 +1,17 @@
+name: "Hello Actions"
```

Comment:
Yaml files are the preferred way to describe structured content.  In this case, the yaml provides interface specification so others using your action know how to interact with it.  You action, however, is not intended to be shared which is also fine.